info-compiler/src/main/java/com/google/sps/webcrawler/WebCrawler.java

```diff
@@ -64,6 +67,15 @@
    *     of permission to access required libraries.
    */
   public WebCrawler() throws IOException {
+    this(DatastoreOptions.getDefaultInstance().getService());
+    this.relevancyChecker = new RelevancyChecker();
```

Comment:
this shouldn't be necessary since the other constructor constructs a relevancy checker as well.